### DISCUSSION TITLE
Beacon Stations lost their "support" (looks weird)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13998] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Beacon Stations lost their decorative "support" at the bottom with the reworks and look like they're floating above the void

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/7665238f-74af-4a24-9bdf-f721ac22f91d)

Before

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/e0a3f5f3-8f3b-4f67-b62d-6baa888b28f1)

After

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/c256d115-31f9-46e7-ab73-15b47337e68d)

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/cbace183-c81a-4be6-bbe1-d457e0cad57e)

---------------

# There's a bigger problem with "floaty" things in general:

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/bf2f193d-5a32-4d93-8b90-3a77d7aeed64)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/6a99e2c9-5af3-4efc-9427-525d19b55482)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/f8bfa16d-e9c6-41e1-8be6-1b2081751b75)

Structures should be able to affect terrain generation around them? 

### Reproduction steps

Generate levels + beacons in leveleditor

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

**Comment by Regalis11** [comment]
Imo this is more of a feature request than a bug report, closing
